dev/tasks/conda-recipes/arrow-cpp/build-pyarrow.sh

```diff
@@ -23,6 +23,7 @@ export PYARROW_WITH_PARQUET=1
 export PYARROW_WITH_PLASMA=1
 export PYARROW_WITH_S3=1
 export PYARROW_CMAKE_GENERATOR=Ninja
+export PYARROW_CMAKE_OPTIONS="-DARROW_SIMD_LEVEL=NONE"
```

Comment:
Hmm, can you add a comment explaining why this is needed?

Reply:
Two things:

- The ppc64le and aarch64 builds somehow try to detect SSE4 and fail as the compiler doesn't support it. There is something wrong with the 
- Otherwise, we probably can set that to SSE2 in the case of x64 but that is probably already part of NONE? conda-forge packages are built for the baseline of CPUs and thus using a higher SIMD level would break them for some users. There is work on supporting instruction set dependent installation in conda but it is not fully done yet.

Reply:
Can you open a JIRA for the ppc64le and aarch64 builds issue? I think this should have been fixed already, so I'm a bit surprised (is cross-compiling involved perhaps?).

NONE really means that we activate no SIMD ISA at compile time. Otherwise you should keep that as DEFAULT.

Reply:
Looking at the values of `ARROW_SIMD_LEVEL` `NONE` is correct for the conda-forge case. We cannot build packages with `SSE4_2` by default as there are still (supported) users without it. I've opened an issue at https://issues.apache.org/jira/browse/ARROW-15445 With the workaround here, we're fine for now but sadly https://issues.apache.org/jira/browse/ARROW-15444 is an issues that should block the release as especially the CUDA builds are used by several downstream users.